### DISCUSSION TITLE
Next Virtual Meeting

### DISCUSSION THREAD

**Discussion by alexanderson1993** [Thorium-Sim/thorium-nova#36] [Q&A] [answered]
Can you believe it's been a week since we first met together to kick off Thorium Nova? Well, there have been a few requests that we have another synchronous meeting.

Personally, I don't think we should rely on these too much, since they make it difficult for everyone to attend and it's hard to keep track of everything that was said without a dedicated notes-taker. Nevertheless, it's still a good way to get lots of ideas out and to answer lots of questions all at once.

So, this discussion is intended to decide when our next meeting is. Post your proposals for what day and time (including timezone) you want the meeting to take place down below as answers to this discussion. Also be sure to include any agenda items you want to cover - I don't want to just meet for the sake of meeting.

If one of the proposals works for you, give it a ⬆️. I will mark the proposal with the most ⬆️ on Thursday morning, which gives us a few days to make proposals and make an announcement via the newsletter. Any ties will be decided by which proposal was made first.

**Edit: Changed the due date to Thursday to better facilitate making an announcement.**

**Comment by astrolamb-gaming** [comment]
I believe an in-person discussion would be appropriate for discussing certain details about [the workings of timelines](https://github.com/Thorium-Sim/thorium-nova/discussions/25#discussioncomment-1461582) and the like. I don't know that anything else has particular need for a meeting at this time.

I propose 7:00 PM EST on Tuesday, Oct. 19th to meet for this discussion. 

Agenda:
- Timeslines and their application, use, and definition from a programming perspective
- Missions vs Timelines, their interactions and differences between each other
- Mission definitions, i.e. programming language vs UI-based (probably just a brainstorming session about this)

Also, I propose we record the discussion for a) future reference, and b) those who may not be able to attend.

**Reply by alexanderson1993** [reply]
Would it be good for me to start writing a discussion about timelines and missions, or should I wait until this meeting happens?

**Reply by astrolamb-gaming** [reply]
Certainly wouldn't hurt

**Reply by Arcadianer** [reply]
Can we maybe try to start at 06:00 EST. This way it would be just midnight in Europe instead of past midnight.

**Comment by alexanderson1993** [comment, marked as answer]
Based on the single response and comments, we'll have a meeting on **Tuesday Oct. 19th at 6:30pm ET**. We'll start half an hour earlier than proposed so @Arcadianer can pop in for a little bit. (I would love to start right at 6:00ET, but that's right when I'm putting Daisy to sleep)

The agenda will be as follows:

- General Q&A and discussion with Alex (if his child is asleep by then 🤞)
- Once @astrolamb-gaming joins, we'll start talking about missions, with @astrolamb-gaming leading the discussion (if that's okay)
  - Timelines and their application, use, and definition from a programming perspective
  - Missions vs Timelines, their interaction, and differences between each other
  - How to define missions, i.e. programming language vs UI-based no-code solution
  
As per the proposals, I will attempt to record the meeting and post it on the [Thorium Sim YouTube Channel](https://www.youtube.com/channel/UC-DADDKagur4-2W5tmhXzFg) shortly after the meeting is over.

Here's the meeting link: https://us06web.zoom.us/j/83973681490?pwd=Y0lyT1ZHeGhHeXZ2OWdBRWlyQ3hnQT09

I'll also send this to newsletter subscribers later today.

I hope to get a Missions/Timeline discussion written up either today or tomorrow that summarizes everything we've already talked about.

**Reply by astrolamb-gaming** [reply]
Honestly the earlier time is better for me as well. I'll happily hop in at 6:15 EST to get started if le babe is asleep by then.

**Reply by alexanderson1993** [reply]
For sure! I mean, the link will let you in whenever you want, so pop in as early as you want, start chatting, and I'll join up as soon as I can.

Just make sure you take notes, just in case the recording doesn't start till I join.
